pkgs/development/libraries/fflas-ffpack/default.nix

```diff
@@ -1,28 +1,35 @@
-{ stdenv, fetchFromGitHub, autoreconfHook, givaro, pkgconfig, openblas
+{ stdenv, fetchFromGitHub, autoreconfHook, givaro, pkgconfig, blas
 , gmpxx
 , optimize ? false # impure
 }:
 stdenv.mkDerivation rec {
   name = "${pname}-${version}";
   pname = "fflas-ffpack";
   version = "2.3.2";
+
   src = fetchFromGitHub {
     owner = "linbox-team";
     repo = "${pname}";
     rev = "v${version}";
     sha256 = "1cqhassj2dny3gx0iywvmnpq8ca0d6m82xl5rz4mb8gaxr2kwddl";
   };
+
   checkInputs = [
     gmpxx
   ];
+
+  enableParallelBuilding = true;
+
   nativeBuildInputs = [
     autoreconfHook
     pkgconfig
   ] ++ stdenv.lib.optionals doCheck checkInputs;
-  buildInputs = [ givaro openblas];
+
+  buildInputs = [ givaro blas ];
+
   configureFlags = [
-    "--with-blas-libs=-lopenblas"
-    "--with-lapack-libs=-lopenblas"
+    "--with-blas-libs=-l${blas.passthru.linkName}"
```

Comment:
Not 100% sure on this but you should be able to just do `blas.linkName`.

Reply:
Indeed